test/Conversion/TorchToTMTensor/basic.mlir

```diff
@@ -0,0 +1,29 @@
+// RUN: torch-mlir-opt <%s -convert-torch-to-tmtensor -split-input-file -verify-diagnostics | FileCheck %s
+
+// CHECK-LABEL: func.func @torch.aten.sort(
+// CHECK-SAME:         %[[INPUT_ARG:.*]]: !torch.vtensor<[3,4],f32>)
+// CHECK:           %[[INPUT_TENSOR:.*]] = torch_c.to_builtin_tensor %[[INPUT_ARG]] : !torch.vtensor<[3,4],f32> -> tensor<3x4xf32>
+// CHECK:           %[[EMPTY_TENSOR:.*]] = tensor.empty() : tensor<3x4xi64>
+// CHECK:           %[[ZERO:.*]] = arith.constant 0 : i64
+// CHECK:           %[[FILL:.*]] = linalg.fill ins(%[[ZERO]] : i64) outs(%[[EMPTY_TENSOR]] : tensor<3x4xi64>) -> tensor<3x4xi64>
+// CHECK:           %[[INDICES_TENSOR:.*]] = linalg.generic {indexing_maps = [#map], iterator_types = ["parallel", "parallel"]} outs(%[[FILL]] : tensor<3x4xi64>) {
+// CHECK:                                           ^bb0(%{{.*}}: i64):
+// CHECK:                                              %[[DIM_1:.*]] = linalg.index 1 : index
+// CHECK:                                              %[[DIM_CAST:.*]] = arith.index_cast %[[DIM_1]] : index to i64
+// CHECK:                                              linalg.yield %[[DIM_CAST]] : i64
+// CHECK:                                    } -> tensor<3x4xi64>
+// CHECK:           %[[SORT:.*]]:2 = tm_tensor.sort dimension(1) outs(%[[INPUT_TENSOR]], %[[INDICES_TENSOR]] : tensor<3x4xf32>, tensor<3x4xi64>) {
+// CHECK:                                   ^bb0(%[[INP1:.*]]: f32, %[[INP2:.*]]: f32, %[[IND1:.*]]: i64, %[[IND2:.*]]: i64):
+// CHECK:                                      %[[PREDICATE:.*]] = arith.cmpf ole, %[[INP1]], %[[INP2]] : f32
+// CHECK:                                      tm_tensor.yield %[[PREDICATE]] : i1
+// CHECK:                            } -> tensor<3x4xf32>, tensor<3x4xi64>
+// CHECK:           %[[OUTPUT_SORTED_TENSOR:.*]] = torch_c.from_builtin_tensor %[[SORT]]#0 : tensor<3x4xf32> -> !torch.vtensor<[3,4],f32>
+// CHECK:           %[[OUTPUT_INDICES_TENSOR:.*]] = torch_c.from_builtin_tensor %[[SORT]]#1 : tensor<3x4xi64> -> !torch.vtensor<[3,4],si64>
+// CHECK:           return %[[OUTPUT_SORTED_TENSOR]], %[[OUTPUT_INDICES_TENSOR]] : !torch.vtensor<[3,4],f32>, !torch.vtensor<[3,4],si64>
+// CHECK:        }
+func.func @torch.aten.sort(%arg0: !torch.vtensor<[3,4],f32>) -> (!torch.vtensor<[3,4],f32>, !torch.vtensor<[3,4],si64>) {
```

Comment:
No need to have this test. See: https://github.com/llvm/torch-mlir/blob/main/docs/architecture.md#dos-and-donts-for-unit-vs-end-to-end-testing

> DON'T use a unit test if your lowering pattern could be described as a trivial "macro expansion" of one op into another op or set of ops. That is, if you feel like your unit test is just rewriting b.create<...>(...) into CHECK: ... then it is probably not a useful unit test.